database/dao.cfc

```diff
@@ -368,7 +368,7 @@ component displayname="DAO" hint="This component is basically a DAO Factory that
 				// would be a closure waiting to be executed.  This will execute and store the PK
 				// value into the table data.
 				if( isClosure( arguments.data[ column ] ) ){
-					 var hydratedColumn = arguments.data[ column ]();

```

Comment:
In this case, arguments.data[ column ] is a closure that needs to be executed to "hydrate" the column with lazy loaded data.  This allows us to only fetch the column's data on demand (important when column may be a relationship that nests other tables/objects)

I had forgotten that Adobe CF doesn't support bracket notated functions so the proper fix for this would be:
```
var hydratedColumn = arguments.data[ column ];
hydratedColumn = hydratedColumn();
```

Reply:
Thanks @chanhong for the PR!  I've been mostly focusing on Lucee so some of these things fell through the cracks!  If you can update your PR with the suggested change I'll merge it!

This also sparked a sweeping clean-up (or, dirty-up in my opinion) to bring the codebase back inline with Adobe CF >= 11.  I had started using member functions, which aren't properly supported until 2016 :(.  

Anyway, good work and thanks for the help!